cpp/subprojects/catch2.wrap

```diff
@@ -0,0 +1,10 @@
+[wrap-file]
```

Comment:
I'm curious about this file... Is it used by Meson?

Reply:
Yes, this is how meson finds and builds external dependencies.